Modules/System/SharePoint/src/model/folder/SharePointFolder.Table.al

```diff
@@ -0,0 +1,81 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Holds information about folder entity.
+/// </summary>
+table 9106 "SharePoint Folder"
+{
+    Access = Public;
+    DataClassification = SystemMetadata;
+    Caption = 'SharePoint Folder';
+    TableType = Temporary;
+    Extensible = false;
+
+    fields
+    {
+        field(1; "Unique Id"; Guid)
```

Comment:
Just call this "Id"

Reply:
Hi, as agreed we will keep the SharePoint naming.